packages/slice-machine/src/components/SideNav/SideNavEnvironmentSelector.tsx

```diff
@@ -87,7 +87,7 @@ export const SideNavEnvironmentSelector: FC<SideNavEnvironmentSelectorProps> = (
           <IconButton
             icon={<LoginIcon className={styles.loginIcon} />}
             onClick={onLogInClick}
-            data-cy="environment-login-icon-button"
+            hiddenLabel="environment login icon button"
```

Comment:
Suggested label change: "Log in to enable environments"

Hidden labels should act as text labels rather than descriptions of the element itself ("button" shouldn't be used in this case).

Reply:
You're 100% right, I forgot the purpose of this with e2e tests!

Reply:
Done